daisy_workflows/kernel_commit_query/kernel_commit_query.wf.json

```diff
@@ -0,0 +1,122 @@
+{
+  "Name": "kernel-commit-query",
+  "DefaultTimeout": "30m",
+  "Vars": {
+    "result_dest": {
+      "Required": true,
+      "Description": "The GCS path for the result data file."
+    },
+    "source_image": {
+      "Required": true,
+      "Description": "The image used to process the kernel commit queries."
+    },
+    "package_manager": {
+      "Required": true,
+      "Description": "The package manager used to install required packages."
+    },
+    "packages": {
+      "Required": true,
+      "Description": "List of packages required to run this workflow."
+    },
+    "pip_packages": {
+      "Required": true,
+      "Description": "List of pip3 packages required to run this workflow."
+    },
```

Comment:
why are these pass-in and required?

Reply:
`packages` is provided to install 1. this program's runtime dependency(that right now consists of git) and 2. the linux distro's kernel source package, now `pip_packages` is only being used to install `google-cloud-storage`.

I strongly agree that `pip_packages` could be hardcoded and not provided as a pass-in.

For `packages` and `package_manager` I understand I could detect the running system's family and infer all the remaining but I like to keep this kind of intel on the caller rather than the workflow itself and keep it as simple as possible (or at least as objective to its goal as possible). 

(PS: They are all required because without the runtime dependencies the program will fail)

Reply:
Got rid of it, see V4 description.